.github/workflows/release.yaml

```diff
@@ -144,17 +144,39 @@ jobs:
         run: |
           unset RELEASE_MINISIGN_PRIVATE_KEY
           unset RELEASE_MINISIGN_PRIVATE_KEY_PASSWORD
-      - name: Set FILES variable from tag
-        run: |
-          echo "FILES=$(find ${{github.workspace}}/.build/release/buf/assets -type f)" >> $GITHUB_ENV
       - name: Release
         id: ghr
         uses: softprops/action-gh-release@v1
+        env:
+          ASSETS_PATH: ${{github.workspace}}/.build/release/buf/assets
         with:
           token: ${{ steps.generate_token.outputs.token }}
           generate_release_notes: true # change to the CHANGELOG somehow
           files: |
-            ${{env.FILES}}
+            ${{env.ASSETS_PATH}}/buf-Darwin-arm64
+            ${{env.ASSETS_PATH}}/buf-Darwin-arm64.tar.gz
+            ${{env.ASSETS_PATH}}/buf-Darwin-x86_64
+            ${{env.ASSETS_PATH}}/buf-Darwin-x86_64.tar.gz
+            ${{env.ASSETS_PATH}}/buf-Linux-aarch64
+            ${{env.ASSETS_PATH}}/buf-Linux-aarch64.tar.gz
+            ${{env.ASSETS_PATH}}/buf-Linux-x86_64
+            ${{env.ASSETS_PATH}}/buf-Linux-x86_64.tar.gz
+            ${{env.ASSETS_PATH}}/buf-Windows-arm64.exe
+            ${{env.ASSETS_PATH}}/buf-Windows-x86_64.exe
+            ${{env.ASSETS_PATH}}/protoc-gen-buf-breaking-Darwin-arm64
+            ${{env.ASSETS_PATH}}/protoc-gen-buf-breaking-Darwin-x86_64
+            ${{env.ASSETS_PATH}}/protoc-gen-buf-breaking-Linux-aarch64
+            ${{env.ASSETS_PATH}}/protoc-gen-buf-breaking-Linux-x86_64
+            ${{env.ASSETS_PATH}}/protoc-gen-buf-breaking-Windows-arm64.exe
+            ${{env.ASSETS_PATH}}/protoc-gen-buf-breaking-Windows-x86_64.exe
+            ${{env.ASSETS_PATH}}/protoc-gen-buf-lint-Darwin-arm64
+            ${{env.ASSETS_PATH}}/protoc-gen-buf-lint-Darwin-x86_64
+            ${{env.ASSETS_PATH}}/protoc-gen-buf-lint-Linux-aarch64
+            ${{env.ASSETS_PATH}}/protoc-gen-buf-lint-Linux-x86_64
+            ${{env.ASSETS_PATH}}/protoc-gen-buf-lint-Windows-arm64.exe
+            ${{env.ASSETS_PATH}}/protoc-gen-buf-lint-Windows-x86_64.exe
+            ${{env.ASSETS_PATH}}/sha256.txt
+            ${{env.ASSETS_PATH}}/sha256.txt.minisig
```

Comment:
It looks like the action supports glob style path patterns, so we might be able to do:

```yaml
files: ${{env.ASSETS_PATH}}/*
```

See https://github.com/softprops/action-gh-release/blob/9114792eb2fcdea3dbb139ffd081fcb856870dcf/src/util.ts#L76-L82 for more details.

Reply:
1 second... will try

Reply:
potentially the definition of overthinking it. resolved in [d4e3f0d](https://github.com/bufbuild/buf/pull/1657/commits/d4e3f0deada832e4dc248961e4392fad3af51f6a)